tests/Samples.Jwt.Tests/EndToEndTests.cs

```diff
@@ -133,7 +133,7 @@ private async Task<string> GetJwtToken()
         return await ProcessOAuthResponseAsync(response);
     }
 
-    private async Task<string> ProcessOAuthResponseAsync(HttpResponseMessage response)
+    private static async Task<string> ProcessOAuthResponseAsync(HttpResponseMessage response)
```

Comment:
There is no reason we should be prompted to mark this as static. This is a test. Performance does not matter and it is highly questionable if performance would be any different. Marking it as static does not help readability. Ditto for many of the rest of the changes

Reply:
> There is no reason we should be prompted to mark this as static.

There is even if this is not so important.

> Marking it as static does not help readability.

This is just a static modifier, come on.